### ISSUE TITLE
Network errors when logging in or uploading pictures with AdAway HOSTS file enabled

### ISSUE THREAD

**Issue by jimbo1qaz** [indywidualny/FaceSlim#218]
Blu R1 HD, Android 6.0.1, AdAway enabled.

When I try to log in, a Google Ads background request fails, and the entire page is replaced by a "domain not resolved" message. Previously, adding a picture would cause the same effect, but I was unable to reproduce.

**Comment by Bahjesaispas** [comment]
Yep adding a picture fail the first time for me too. But if you go back and try again it works. One+ one adaway installed on 6.0.1

**Comment by bwstefano** [comment]
I can confirm this behavior. I've been using DNS66, an app that acts the same way as AdAway, and I've randomly seen those background requests being blocked. So far I saw requests within FaceSlim to Google Ads and imrworldwide.com.
